integration-tests/logging-panache/pom.xml

```diff
@@ -30,6 +30,11 @@
             <artifactId>quarkus-jdbc-h2</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-agroal-dev</artifactId>
+            <scope>test</scope>
```

Comment:
Why is this test dependency added here? I can't see any other file or change in this module. Maybe it deserves a comment above the dependency for others to understand why is it present.

Reply:
This dependency was removed in a follow up PR https://github.com/quarkusio/quarkus/commit/a84b2852c44480da970a782c2dccda0a02e9a485

Reply:
Alright, thank you.